artemis-server/src/main/java/org/apache/activemq/artemis/core/server/impl/FileLockNodeManager.java

```diff
@@ -267,25 +293,27 @@ public final SimpleString readNodeId() throws ActiveMQIllegalStateException, IOE
       return getNodeId();
    }
 
-   protected FileLock tryLock(final int lockPos) throws Exception {
+   protected FileLock tryLock(final long lockPos) throws IOException {
```

Comment:
lock is changed from int to long here, was there a need? 

if so other methods such as lock should be updated, if not this should be reverted

Reply:
java.nio.channels.FileChannel#tryLock takes longs so that's why I changed it.  I updated the rest of the methods/variables to be consistent.

Reply:
thanks for explanation, and +1 for consistency cheers.

Reply:
+1 on this.  Merging.